### ISSUE TITLE
Diagram scrollMargin parameter bug

### ISSUE THREAD

**Issue by aszokalski** [NorthwoodsSoftware/gojs-react#10]
Hi! I think I found a bug in gojs-react. 
Im trying to run my gojs app using React and Electron. During my rewrite I noticed that `scrollMargin` parameter in 'go.Diagram' was causing the following errors:

```
go.js:773 Uncaught TypeError: this.Yq is not a function
    at Q.set (go.js:773)
    at zj (go.js:651)
    at Ll (go.js:929)
    at Jl (go.js:924)
    at Object.initDiagram (DiagramWrapper.tsx:70)
    at ReactDiagram../node_modules/gojs-react/lib/esm/gojsreact.js.ReactDiagram.componentDidMount (gojsreact.js:23)
    at commitLifeCycles (react-dom.development.js:22101)
    at commitLayoutEffects (react-dom.development.js:25344)
    at HTMLUnknownElement.callCallback (react-dom.development.js:336)
    at Object.invokeGuardedCallbackDev (react-dom.development.js:385)
```

and 

```
react-dom.development.js:12212 Uncaught TypeError: this.Yq is not a function
    at Q.set (go.js:773)
    at zj (go.js:651)
    at Ll (go.js:929)
    at Jl (go.js:924)
    at Object.initDiagram (DiagramWrapper.tsx:70)
    at ReactDiagram../node_modules/gojs-react/lib/esm/gojsreact.js.ReactDiagram.componentDidMount (gojsreact.js:23)
    at commitLifeCycles (react-dom.development.js:22101)
    at commitLayoutEffects (react-dom.development.js:25344)
    at HTMLUnknownElement.callCallback (react-dom.development.js:336)
    at Object.invokeGuardedCallbackDev (react-dom.development.js:385)
```
And NOTHING shows on the screen. Even stuff unrelated to GoJS.
It's something about this 'Yq' function. I haven't looked into it deeper. Here's my code for reference:
- Not working:
```
const diagram =
      $(go.Diagram,
        {
            scrollMargin: 2000,
            [other params]
        });

```

- Working:
```
const diagram =
      $(go.Diagram,
        {
            [other params]
        });

```

**FIX**
For anyone having this problem: only removing this parameter worked for me so that's I guess what you have to do until it's fixed if you want your app to show anything.

If anyone knows a workaround without tradeoffs please share.

**Comment by WalterNorthwoods** [comment]
Precisely which version of GoJS are you using?  Please try 2.1.31.

**Comment by aszokalski** [comment]
Hi Walter. I already am:   
"gojs": "^2.1.31",
    "gojs-react": "^1.0.10"

**Comment by simonsarris** [comment]
This looks like a bug in GoJS, we'll fix it for the next release. Thank you for reporting.

**Comment by simonsarris** [comment]
This has been fixed and will be out with the next release, probably within a week or so. I do not have a workaround however, other than setting `scrollMargin` later, after the Diagram has a DIV associated with it, or possibly this might work also as a workaround:

```
const diagram =
      $(go.Diagram,
        {
            scrollMargin: 2000,
            [other params]
        });
diagram.redrawLater = function() {};
```

But should be removed with the next release.

**Comment by simonsarris** [comment]
This is now fixed with the release of GoJS 2.1.32. You should not need the workaround any more.
